docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -10,11 +10,36 @@ default:: `syn-vertical-pod-autoscaler`
 
 The namespace in which to deploy this component.
 
-
 == `images`
 
 [horizontal]
 type:: dictionary
 default:: https://github.com/projectsyn/component-vertical-pod-autoscaler/blob/master/class/defaults.yml[See class/defaults.yml]
 
 A dictionary of images used by this component.
+
+== `autoscaler`
+
+[horizontal]
+type:: string
+default:: {}
+
+Create autoscaler resources.
+
```

Comment:
Please add a quick explanation how the key is used and what fields are available.

Reply:
I added an example under that topic.